modules/serverless-services-network-policies.adoc

```diff
@@ -0,0 +1,82 @@
+[id="serverless-services-network-policies_{context}"]
+= Enabling communication with Knative applications on a cluster with restrictive network policies
+
+If you are using a cluster which multiple users have access to, your cluster might use network policies to control which pods, services, and namespaces can communicate with each other over the network
+
+If your cluster uses restrictive network policies, it is possible that Knative system pods are not able to access your Knative application. For example, if your namespace has the following network policy, which denies all requests, Knative system pods cannot access your Knative application:
+
+.Example NetworkPolicy object that denies all requests to the namespace
+[source,yaml]
+----
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: deny-by-default
+  namespace: example-namespace
+spec:
+  podSelector:
+  ingress: []
+...
```

Comment:
@abrennan89, So for a default deny policy, there is no 'other' YAML that might be included. There's no need for `...` here.

Reply:
Thanks, Jason!